packages/api/src/utils/index.ts

```diff
@@ -25,6 +30,50 @@ export function createFeedFullName (network, name, decimals) {
   return `${network}_${name.split('/').join('-')}_${decimals}`
 }
 
+// normalize config to fit network schema
+
+export function normalizeNetworkConfig (
+  config: RouterDataFeedsConfig
+): Array<NetworksConfig> {
+  // get list of chains
+
+  const chains: Array<string> = Object.keys(config.chains)
+
+  // get list of networks
+  const networks: any = Object.values(config.chains).map(chain => {
+    const networks = Object.values(chain.networks).map((value, index) => {
+      return {
+        key: Object.keys(chain.networks)
```

Comment:
Can we move this to a function and use it [here](https://github.com/witnet/data-feeds-explorer/pull/148/files#diff-ff98f56bb76f03b90aaaa0aabf9203a8666746824c998d36cbce775699b48fe7R5)?